.gitignore

```diff
@@ -0,0 +1,2 @@
+.terraform
```

Comment:
I had to [check](https://git-scm.com/docs/gitignore#_pattern_format), but yes: this will match the folder (not just a file named that):

> If there is a separator at the end of the pattern then the pattern will only match directories, otherwise the pattern can match both files and directories.